sdk/contentsafety/ai-content-safety-rest/review/ai-content-safety.api.md

```diff
@@ -100,23 +100,22 @@ export interface AnalyzeImageDefaultResponse extends HttpResponse {
 export interface AnalyzeImageOptions {
     categories?: string[];
     image: ImageData_2;
+    outputType?: string;
```

Comment:
@jeremymeng I remember this `ImageData_2` issue is fixed, do you have any idea this issue is back again?

I believe we only have one ImageData defined.

Reply:
@MaryGao It's the annoying api-extractor behavior and I don't remember we ever get to the bottom of this.